.travis.yml

```diff
@@ -73,6 +73,10 @@ jobs:
         - make go-test
         - make mod-check || travis_terminate 1;
 
+    - name: containerized test
+      script:
+        - make containerized-test || travis_terminate 1;
```

Comment:
can we have this with the containerized build itself? spanning more jobs takes lot of time when we have multiple builds in flight

Reply:
sure, I can combine them in a single job

Reply:
done!